core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalCheckpointThread.java

```diff
@@ -228,6 +240,7 @@ private void runInternal() {
         maybeCheckpoint();
         if (mShutdownInitiated) {
           if (quietPeriodWaited || !mWaitQuietPeriod) {
+            mReplayState = ReplayState.REPLAY_DONE;
```

Comment:
If the tailer is shutdown, then this means it is already being signaled to become primary from secondary? Is that transition when we want to set the replay state to done?

Reply:
I don't think this line actually do anything. I add it to be safe.
When initial replaying, the master will not be able to gainPrimacy() or losePrimacy() to shutdown the checkpoint thread. If the AlluxioMaster itself is shutting down, replay is done or not doesn't make a difference.